README.md

```
@@ -1,36 +1,125 @@
+- **E2E Tests**: Playwright tests for the full edit purchase flow
+
+Run all tests:
+```bash
+pnpm test:run    # Unit tests
+pnpm test:e2e    # E2E tests
+pnpm test:e2e:ui # E2E tests with Playwright UI
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)